apps/designer/app/designer/designer.tsx

```diff
@@ -219,7 +219,7 @@ const NavigatorPanel = ({ publish, isPreviewMode }: NavigatorPanelProps) => {
 
 type DesignerProps = {
   config: Config;
-  project: Project;
+  project: db.project.Project;
```

Comment:
btw this is a total workaround, we need to have a way to get the augmented Project type, which is currently not importable in a good way, we need a project package I think

Reply:
Might be better to have a single package that works with database. Its API would be based on the augmented types, and as a user of this package I would know nothing about `postgres` or maybe even `prisma`. 

But it should handle all database objects. Otherwise I feel like we'll end up in a dependency hell and might even have to deal with circular NPM dependencies which I don't think possible.

Reply:
The problem with this centralized approach is that it will end up depending on a LOT of other types across the application.

There is a another way - shared functionality between features always goes into a shared folder that is named based on its function/scope.

So for example there would be a shared/project folder that has everything about the project including augmented database types and utilities.

Reply:
The way it is right now is not great, because we tried initially to centralize database stuff in a shared/db folder and this is currently the result.

Reply:
Ultimately if we have it this way no feature will have to access prisma client directly, they will all work with shared/project shared/instance etc, which in return will call into prisma and augment data and types